docs/root/intro/version_history.rst

```diff
@@ -8,6 +8,9 @@ Version history
 * admin: added config dump support for Secret Discovery Service :ref:`SecretConfigDump <envoy_api_msg_admin.v2alpha.SecretsConfigDump>`.
 * buffer filter: the buffer filter populates content-length header if not present, behavior can be disabled using the runtime feature `envoy.reloadable_features.buffer_filter_populate_content_length`.
 * config: added access log :ref:`extension filter<envoy_api_field_config.filter.accesslog.v2.AccessLogFilter.extension_filter>`.
+* config: added support for :option:`--reject-unknown-fields-dynamic`, providing independent control
+  over whether unknown fields are rejected in static and dynamic configuration. By default, unknown
+  fields in static configuration are rejected and are allowed in dynamic configuration.
```

Comment:
Can you mention that there are stats for both of these cases and link to their location? Also mention warnings, etc.? I think this is really important from an operations perspective and we should let folks know what the options are. I would also mention these things in cli.rst as well.